test/runtests.jl

```diff
@@ -15,8 +15,7 @@ test_infile = @__FILE__
 test_compressed = joinpath(tmp, "runtests.jl.gz")
 test_empty = joinpath(tmp, "empty.jl.gz")
 
-@windows_only gunzip="gunzip.exe"
-@unix_only    gunzip="gunzip"
+gunzip = "gunzip" * (is_windows() ? ".exe" : "")
```

Comment:
Clever.

Not a big deal if you choose not to, but I think this is clearer written closer to the original way.